src/games/strategy/triplea/ai/proAI/util/ProMatches.java

```diff
@@ -147,18 +147,6 @@ public boolean match(final Territory t) {
     };
   }
 
-  public static Match<Territory> territoryCanMoveLandUnitsAndIsEnemy(final PlayerID player, final GameData data) {
```

Comment:
We can remove these couple of matches as I think I originally was using them and then refactored but didn't delete them.